topology/University of Florida/UF HPC/UFlorida-HPC.yaml

```diff
@@ -128,10 +128,6 @@ Resources:
         Primary:
           ID: 2d9012f731159dd45a082164c6aac6577167e725
           Name: Bockjoo Kim
-      Submitter Contact:
-        Primary:
-          ID: dcc5225ec8889c92053920bbd934bedadc799908
-          Name: Craig Prescott
```

Comment:
Why didn't we replace this with the mailing list?

Reply:
The Submitter is the person who originally submitted the resource, and we can't change that without a time machine :)

He doesn't work there though so there's no point in keeping the entry.

Reply:
Eh, we have plenty of old submitter info throughout the repos. Let's just keep it around until we decide on the fate of the submitter contacts as a whole.